smarttub/api.py

```diff
@@ -465,6 +465,10 @@ def __init__(self, spa: Spa, **properties):
 
         self.intensity = properties["intensity"]
         self.mode = self.LightMode[properties["mode"]]
+        try:
+            self.cycleSpeed = properties["cycleSpeed"]
+        except KeyError as e:
+            logger.debug(f'Benign: {e}')
```

Comment:
```suggestion
        self.cycleSpeed = properties.get("cycleSpeed", None)
```

This way, the code below which accesses `self.cycleSpeed` won't throw an exception when it wasn't set